src/main/java/org/opentripplanner/standalone/config/RouterConfig.java

```diff
@@ -43,10 +45,7 @@ public RouterConfig(JsonNode node, String source, boolean logUnusedParams) {
         );
         this.transitConfig = new TransitRoutingConfig(adapter.path("transit"));
         this.routingRequestDefaults = mapRoutingRequest(adapter.path("routingDefaults"));
-
-        // Touch parameters read at a later point in time to avoid reporting them when
-        // logging unused parameters.
-        adapter.path("updaters");
+        this.updaterConfig = new UpdaterConfig(adapter.asRawNode("updaters"));
```

Comment:
Use the `path` method, and `asRawNode()`and pass in 2 arguments (if you need access to the JsonNode). I will explain below.

Reply:
I don't think you explained how to do this

Reply:
I removed this method, as it is not needed anymore.